package/MDAnalysis/core/AtomGroup.py

```diff
@@ -466,14 +463,19 @@ class Atom(object):
     """
 
     __slots__ = (
-        "index", "id", "name", "type", "resname", "resid", "segid",
-        "mass", "charge", "residue", "segment",
-        "_universe",
-        "radius", "bfactor", "resnum", "serial", "altLoc")
+        "index", "id", "name", "type", "resname", "resid", "segid", "mass",
+        "charge", "residue", "segment", "_universe", "radius", "bfactor",
+        "resnum", "serial", "altLoc")
 
     def __init__(self, index, name, type, resname, resid, segid, mass, charge,
-                 residue=None, segment=None, radius=None, bfactor=None,
-                 resnum=None, serial=None, altLoc=None, universe=None):
+                 residue=None,
```

Comment:
This is always done by 'yapf' for named arguments. I currently  don't know if changing the default helps to prevent this.